### PR TITLE
Added Customization UG and images for MAUI PullToRefesh control

### PR DESCRIPTION
**Pull request by DiwakarVenkatesan**
Added Customization UG and images for MAUI PullToRefesh control

### CODE REVIEW

File: MAUI/Pull-to-Refresh/customization.md [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+## PullableContent
+
+The `PullableContent` is the main view of the `PullToRefresh` control on which the desired items can be placed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="6 11" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh"
+                                PullingThreshold="120"
+                                RefreshContentHeight="30"
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/customization.md:23]: Check the API names

File: MAUI/Pull-to-Refresh/customization.md [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Host .NET MAUI DataGrid as pullable content
+
+The `PullToRefresh` control provides support for loading any custom control as pullable content. To host the .NET MAUI Datagrid inside the PullToRefresh, follow these steps.
+<ol>
+    <li> Add the required assembly references to the pcl and renderer projects as discussed in the Assembly deployment section of <a href="https://help.syncfusion.com/maui/datagrid/getting-started">DataGrid </a> and <a href="https://help.syncfusion.com/maui/pulltorefresh/getting-started">PullToRefresh </a>.</li>
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/customization.md:264]: check this content and change it to MAUI standard.

Add the required assembly references as discussed in the getting-started section of ...

File: MAUI/Pull-to-Refresh/customization.md [hunk 3]
```diff
@@ -0,0 +1,640 @@
...
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    pullToRefresh.TransitionMode = PullToRefreshTransitionType.SlideOnTop;
+
+{% endhighlight %}
+{% endtabs %}
+
+![PullToRefresh with slide on top transition mode](Images/customization/maui-pulltorefresh-getting-started.png)
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/customization.md:60]: follow standard for Alt text
![Slide-on-top transition mode in .NET MAUI PullToRefresh] 
(or)
![.NET MAUI PullToRefresh with slide-on-top transition mode]

check the whole document for this comment.

File: MAUI/Pull-to-Refresh/customization.md [hunk 4]
```diff
@@ -0,0 +1,640 @@
...
+        }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![PullToRefresh view Template](Images/customization/template-slideontop.gif)
+
+You can download the sample code by clicking the following link: [Sample](https://github.com/SyncfusionExamples/load-datagrid-as-pullable-content-of-.net-maui-pull-to-refresh).
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/customization.md:640]: add the sample link in Notes like below (Refer Expander GettingStarted UG]

image

File: MAUI/Pull-to-Refresh/customization.md [hunk 5]
```diff
@@ -0,0 +1,640 @@
+---
+layout: post
+title: Customization in .NET MAUI PullToRefresh control | Syncfusion
+description: Learn about Customization support in Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control and more.
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/customization.md:4]: Learn about Customization features in ....

File: MAUI/Pull-to-Refresh/customization.md [hunk 6]
```diff
@@ -0,0 +1,640 @@
+---
+layout: post
+title: Customization in .NET MAUI PullToRefresh control | Syncfusion
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/customization.md:3]: Customization features in ..

**Review by PiruthivirajM — CHANGES_REQUESTED** [review]
Please address the review comments.

File: MAUI/Pull-to-Refresh/customization.md (hunk 1 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/customization.md:23]: changed

File: MAUI/Pull-to-Refresh/customization.md (hunk 2 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/customization.md:264]: changed the content

File: MAUI/Pull-to-Refresh/customization.md (hunk 4 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/customization.md:640]: changed

File: MAUI/Pull-to-Refresh/customization.md [hunk 7]
```diff
@@ -0,0 +1,640 @@
...
+If you run the above sample with the TransitionMode as Push, the output will look as follows.
+
+![PullToRefresh with DataGrid hosted with push transition mode](Images/customization/datagrid-push.gif)
+
+
+## Host .NET MAUI ListView as pullable content
+
+To host the .NET MAUI `ListView` inside the `PullToRefresh`, which is used to update items in the list while performing the pull to refresh action.
+<ol>
+    <li>	Add the required assembly references to the pcl and renderer projects as discussed in the Assembly deployment section of <a href="https://help.syncfusion.com/maui/listview/getting-started">ListView </a> and <a href="https://help.syncfusion.com/maui/pulltorefresh/getting-started">PullToRefresh </a>.</li>
```
[the remark this answers is not in the document]
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/customization.md:380]: changed

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**